media-libs/ctl/ctl-1.5.2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="AMPAS' Color Transformation Language"
+HOMEPAGE="https://github.com/ampas/CTL"
+SRC_URI="${HOMEPAGE}/archive/ctl-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AMPAS"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+
+RDEPEND="media-libs/ilmbase:=
+	 media-libs/openexr:=
+	 media-libs/tiff:=
+	 !!media-libs/openexr_ctl"
```

Comment:
why the !! block instead of simple ! ?

Reply:
When I did testing on my system, I found this was the only way to force the uninstall of openexr_ctl. Maybe I had something broken at the time, but never came back to revisit this. I can take a look now and see if it's still needed.

Reply:
If you just want to prohibit both to be installed at the same time, you should use !
If you want to prohibit openexr_ctl being present when building ctl, you should use !!

I think you want the former, as the latter is for very rare cases when you can't even build ctl if openexr_ctl is installed: this about a build host generating binpkgs that will not install the package for example

Reply:
Ah. Very good information. I'll change it tomorrow.